nginx/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# a minimal Nginx container including containerbuddy and a simple virtualhost config
```

Comment:
To which I should add, if I wanted a series of pluggable applications that I could treat like "Docker libraries" they'd all be in the same repo.